lib/engines/content_object_store/app/views/content_object_store/content_block_editions/edit.html.erb

```diff
@@ -0,0 +1,51 @@
+<% content_for :context, "Object store" %>
+<% content_for :title, "Change #{@schema.name}" %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: content_object_store.content_object_store_content_block_edition_path(@content_block_edition),
+  } %>
+<% end %>
+
+<%= form_with model: @content_block_edition, as: :content_block_edition, url: content_object_store.content_object_store_content_block_edition_path do |f| %>
+  <%= hidden_field_tag "content_block_edition[content_block_document_attributes][block_type]",
+    @schema.block_type,
+    id: "content_object_store/content_block_edition_content_block_document_block_type" %>
+
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Title",
+    },
+    name: "content_block_edition[content_block_document_attributes][title]",
+    id: "content_object_store/content_block_edition_content_block_document_title",
+    value: @content_block_edition.content_block_document&.title,
+    error_items: errors_for(@content_block_edition.errors, "content_block_document.title".to_sym),
+  } %>
+
+  <% @content_block_edition.details.each do |field, value| %>
```

Comment:
This block is not in _form and so is the main difference.

We could attempt to partial up other parts of the page but it feels like too much indirection just to dry things up.